website/docs/r/project.html.markdown

```diff
@@ -0,0 +1,159 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_project"
+description: |-
+  Manages project.
+subcategory: "Projects API Specification"
+---
+
+# ibm_project
+
+Provides a resource for project. This allows project to be created, updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_project" "project_instance" {
+  configs {
+		id = "id"
```

Comment:
Correct me if I'm wrong here. 
id here is project id? If so how do we provide project id during creation of project.